Makefile

```diff
@@ -3,6 +3,9 @@ EPOCH_TEST_COMMIT ?= f2925f58acc259c4b894353f5fc404bdeb40028e
 PROJECT := github.com/kubernetes-incubator/cri-containerd
 BINDIR ?= ${DESTDIR}/usr/local/bin
 BUILD_DIR ?= _output
+# VERSION is the version of the binary.
+# TODO: push a tag so this will not print a fatal error
+VERSION:=$(shell git describe --tags --dirty)
```

Comment:
Kubernetes is using similar solution, but more complex https://github.com/kubernetes/kubernetes/blob/master/hack/lib/version.sh.

Actually, once we have a tag, this will work well. The problem is that I'm not sure what tag to create given that we don't have the first version.